### PR TITLE
ACM-2869

### PR DESCRIPTION
**Pull request by chenz4027**
Signed-off-by: Maggie Chen <magchen@redhat.com>
https://issues.redhat.com/browse/ACM-2869

### CODE REVIEW

**Comment by chenz4027** [comment]
@jeswanke the fourth files doesn't look right to me... will take a look tomorrow

File: frontend/src/components/CreateCredentialModal.tsx [hunk 1]
```diff
@@ -4,16 +4,28 @@ import { Fragment } from 'react'
...
     <Fragment>
       <ContextSelectorFooter>
-        <Button onClick={props.handleModalToggle} variant="link" isInline>
+        <Button
+          onClick={
+            control
+              ? () => {
+                  handleModalToggle(control)
+                }
+              : handleModalToggle
```
Comment (KevinFCormier) [inline frontend/src/components/CreateCredentialModal.tsx:24]: You can just pass this function:
```
() => {
  handleModalToggle(control)
}
 ```
 
 If the function doesn't use affectedControl param, passing the argument won't hurt.

**Pull request merged**